doc/notebooks/jupyter_integration.ipynb

```diff
@@ -0,0 +1,155 @@
44GnCI0cOA+tUFn+Mc9ckqsfYTUkixJqK87GTE4C/r4qFp7nmdUpUQTR14/a8b3jqcoJo3xmkgx3YIN/S3LY74GF7y8RcfJqICaK5tyMTqOFcU+6+6Du5k5Zr5i7JaKlbM3ZH2qUNviu8KBDGGtYaq+uQ2BUVIkqI+z63Bz64hnCuvk5XXb4yKqFpdvmolH0QWbcoop07PLMBthUWToEzpAMB4ti1eyKvk/Ti/lFq3Nr9A2bcg3M7e+KVZs0ArZ37ocO30ZI2eYDWAUXng+NDnxS7ophd3SjEruiL0MknNTg0+kKAhTpsG+qJrizLhJRWRrNXjMopVUQ3OOAzeAN/DJ+Jazxqn4I/6n5Fc7cCi+xb1RPU4owWf1T5zla4tRVojCrRY/d1GxHqGXBjU7dkd9cpy44wsY4uwn0VybpLFkxaqvUSPpONRMjFSWTzVsFY4Sn8BH4O/4fghEd+PHoHHPT07kWk0naJx2JDaTRlLw0sZ3aRcnh9HPYrnyj++U0ZMB3MslMQ1WzZTiq7J1ixfHRh0hQmV/hbGExN11LSzRjZpjxpykneEDzfLwY2fu3zkThRS1gYMsPCYF/UpshZ2l2hqLkzaNJf1NT3TGzjdAqRsza6P6Xe+mKp6iX3K8P5gRTlxsGJrCPrjrOpaiQB3b7+wa+uAQ/RM60ysOw4STk56B56NtlMXXWh5giseSEOpIlUqZWyUlaKl8knluYqkjLyQrrjcrp56/IEbRBakJmrXdnvNcM79+vmYHaci/f4ZvtHxVL2e20D+FI3KmpkGeLqrpRGFDGr0Ob7Ws4T2gwD+9k0NDJcENcS58JK0CApuXBN211JIW926GuxcQw1QZjqU02CfQe7xmkK6iNa1UpUEjkcKnt4T1qN/u0jFG2Yh+2ZVbV5hUo+l2JmGo9LPGus9SQP13xko4y1eTWGN15qSGLpSLa166KJAPvKGjwvff6M8F5PZQu+iyLXSaXxlk8Puu/NNyvQOcIhsRqw+ZVUn6pcZtTvLbdeO0y1GZ+WiOdua+mxelAkXuXC0Wm0fp7mtJKC8tbAfUs8eNq/qvXsFt7pKj+Gl81eDdrtJ3nNzDeQ4C38HD6AfwZfw6nrrDa/9TcHfu8tXKptqChgPYrTasdebVq1c8N2LYN8Kh0ovaiIFuFEaEz1RYB27m7N0M11gqJ5hW3YoylKrtICL4rinsJfwKfwE3gHb/S4Z318KIyVjxN38CV8Cv83fPoDe//YjkfvgO0xuxVvYuiaQY8UT63ceC1MBsMm8vB65Rn8W/jJr6NA95+PMJAF3kos9Z66hiaPYIRVM08mK1EhkVfGoFF95vm8nXfoLMKFDFCVkKvB3X+lYsUdnGhUsgbYWq20N4thM69mJaMxkxtPVUTfGhDtFlAqUzUoyebY3HOx+CzewAO5hjKQD9zvOYnSnJsgg9dvgMGXeuIcGDM1knfMl9yHWHPxST4DubJWKM629WS61ewH1tNSw/4pIVFLLWss88QycR5Zd5ySq1hniwl2nAeWQjp+cQjU/R+cKBKwX3RP9/p9gJdQGd6vlrJXAgfyMVkXbFB0JnWqB3T6Tv+5DKweqDXgZICElD1mwES+elo+1C0nKx2HOTCsqwh9EQUHNr852LtSVzK0NW+za6qSjw/UGnvnlKIGuVdLfOPiza9Xnv+1skKMhUJjaak64C3vqBKBXwPnshtu15TzVs21wq3FeanT4S3wRCXYqmClaBGyHOpeGdWRqwu+ggrfwstrQoA7JXepIzCPelrv5OQG3Q46vtUORlY4DZzg2estkrbb+X0Ho0qNA+ONzNGg0Tt4Jqebuuac3CXELUGsirfOHWG7vcbeaEjsE0hexbZpTP8KfgofwVP4MyBTGmMLuIbbrmsYSK6Z9t/DFdzBlYQtjRV6q11zhN97Bt/BU01Qsk++Eo4yKqlsBYLSZfaDQo2igKPqetssaoXmtF6pope12kSCyZOOdwIFDJiY1Rz+RmycoiGc78QCeu8NJd7gWRVTjZGUGeApPIH/AX4X3vxmw/9IjkfvgG2T7eEdvBeDcfHuI7ZQPZh8j4kYrwwM8JX4+P8BBzx0NtE2xXs4ktfUlKc06ifO7Mwj2lsLrCQjtoysxvvdTLOd/F57xNrPrzXc5K30gIIBhYO0IQeVTnuyUmpIrywjVrU9MwFPuDG42Ly4OdqNQxtbDuq6EAeO91w0iw906W8tzme2yCZlkmVIQJyKJy4JJvTvtZCC9LzCvnKmMaJtcOyZnWYGO6g+EgdJPM5MJS9h2BEiIVqOvuc0skws5uMHH/3kU4mG/WlTMtqratQ0T7IyQsuPT+6c4lKk3tByVg9cqv6xIc8Wc7Sus4HVI3e7uU9kPxJD9p6r6t7UM/7YhhYk2XFklYCJMFs1Qdql0seIoTwQmOz8vZMbqsoldzJR80Y8HlbnssVYfGrQTnXHPfFYagnhouoJqwBnFz01LF1jtpskZzT+fCVwJFJCrlhl12z8lcq3zwT2tILoQagJnbezX6J8MFD5NvBEJN4PdiLetHAnqLd+7CDTloYe9IgBA6fR7/kJXq1an6AlbXBrO5mqtC92tZtWTr6VvkeAd533tfvYNC+rkrKiEQK5q/4G2ZykEEFVqktd5SwPaq6ZE2RiZAumm2FBn3/Pq8ifD3wuRRYTz/r38LvwFj5UqPPf2gcaCe4a3so1tgen5fpF1qkqwbSFamzwWQFK0L+/hWsR+EMn3tCKDv163nF5wRJcxKPCa/n3DG/hFp7ACV7DPXwjbtpJ7jnJpjYdsx1cJH/i/0iDb34oQPzYjkfvgO24knfkIZ6zCd9jbsksyMow305TmFmkiPriNwDRl3ADdxJEPznEXeXI7dOAVewhQ2ujBv81mHHH2ZVbU2dfGu5kGN1Lha6zmthvYKYQE6X6YN0grpAzrtdu1EG7CHMbC5PlW9e8t2wVCLHWEmhiBSMxlFSztVHtDDMlWJ90JQTvI4qZYWQx7a2zf3IZGWcmizB283m
```

Comment:
Let's follow the policy of having un-computed notebooks in this repo. The point is that you can compute them easily, right? They are different from the standard documentation (where we have images) and from the notebooks linked from the website which should really look as nice as possible, so they include images.